templates/v2/reset_password/beneficiary/secret_answer.html.twig

```diff
@@ -0,0 +1,9 @@
+{% extends 'v2/reset_password/beneficiary/reset_password_form_layout.html.twig' %}
+
+{% block formTitle %}
+    <h4 class="text-center mt-4">{{ beneficiary.questionSecrete }}</h4>
+{% endblock %}
+
+{% block form %}
+    {{ include('v2/reset_password/beneficiary/_form.html.twig', {formField: form.secretAnswer}) }}
```

Comment:
You could also pass the form title in here no ? It would require a small blocks refactoring